CreatubblesAPIClient/Sources/CreationUploadSession/CreationUploadSession.swift

```diff
@@ -130,21 +131,44 @@ class CreationUploadSession: NSObject, Cancelable
                             weakSelf.delegate?.creationUploadSessionChangedState(weakSelf)
                             weakSelf.notifyServer(error, completion: { (error) -> Void in
                                 
-                                weakSelf.error = error
-                                weakSelf.isActive = false
                                 weakSelf.delegate?.creationUploadSessionChangedState(weakSelf)
                                 
-                                if let error = error
+                                if let _ = weakSelf.creationData.galleryId
```

Comment:
If you guard for galleryId in `uploadToGallery(...)` you will be able to remove if/else from here